### DISCUSSION TITLE
GitHub stopped sending me email updates

### DISCUSSION THREAD

**Discussion by dothackjhe** [community/community#78519] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I'm not sure what's causing this and for how long already, but I no longer receive email updates from GitHub for some reason.

**Comment by Kiran1689** [comment]
Hi @dothackjhe 

Please check your settings/Notification and settings/email and Ensure that your email address is correctly configured in your GitHub account settings.
